### ISSUE TITLE
Selection paste does not work in konsole

### ISSUE THREAD

**Issue by seletskiy** [reconquest/clipboard-syncd#4]
How to reproduce:

1. Start xsyncd daemon;
1. Run two instances of konsole;
1. Select text in one instance of konsole;
1. Try to paste (`Shift+Insert`) it into second instance of konsole;
* Selected text will not be inserted;

Selection which was made in different application (e.g. firefox) pastes successfully.

If selection is modified directly via `xclip` it also pastes successfully, e.g.:

```
xclip -i -selection clipboard <<< TEST
```

Relevant DEBUG log:
```
main.c:174 DEBUG:UTF8_STRING = 0x149
main.c:174 DEBUG:STRING = 0x1f
main.c:174 DEBUG:XSEL_DATA = 0x180
main.c:174 DEBUG:INCR = 0x181
main.c:174 DEBUG:TIMESTAMP = 0x182
main.c:174 DEBUG:TARGETS = 0x183
main.c:174 DEBUG:MULTIPLE = 0x184
main.c:174 DEBUG:TEXT = 0x185
main.c:174 DEBUG:PRIMARY = 0x1
main.c:174 DEBUG:CLIPBOARD = 0x186
main.c:564 DEBUG:Got an event: (null) (87)
main.c:526 DEBUG:handling xfixes_selection_notify
main.c:534 DEBUG:Selection: 0x1
main.c:564 DEBUG:Got an event: PropertyNotify (28)
main.c:548 DEBUG:handling property_notify
main.c:551 DEBUG:XSEL_DATA property changed
main.c:564 DEBUG:Got an event: (null) (159)
main.c:507 DEBUG:handling selection_notify
main.c:330 DEBUG:Selection: 0x1; Sync selection: 0x186
main.c:353 DEBUG:Freeing old clipboard data: (null)
main.c:358 DEBUG:New clipboard data is: XXX

main.c:564 DEBUG:Got an event: (null) (87)
main.c:526 DEBUG:handling xfixes_selection_notify
main.c:564 DEBUG:Got an event: PropertyNotify (28)
main.c:548 DEBUG:handling property_notify
main.c:551 DEBUG:XSEL_DATA property changed
main.c:564 DEBUG:Got an event: SelectionRequest (30)
main.c:438 DEBUG:handling selection_request
main.c:489 DEBUG:We don't handle this target yet (0x183).
main.c:564 DEBUG:Got an event: SelectionRequest (30)
main.c:438 DEBUG:handling selection_request
main.c:489 DEBUG:We don't handle this target yet (0x183).
```

**Comment by RomanKovtyukh** [comment]
Hello, Mr. Meeseeks(@seletskiy ), thank you for your feedback.

I see what it the problem you have. This is because of unimplemented feature. I implemented it now, but, unfortunately I can't find any applications on my machine to test this implementation.

Could you please tell me about the terminal emulator you are using to make me able to test this situation?

**Comment by RomanKovtyukh** [comment]
It seems that we need to write a simple util for functional tests, that will be on lower level than xsel or xclip and will do such requests from sync daemon, because now I can't use xsel to write simple test case for this situation.

**Comment by seletskiy** [comment]
@Grandmother: hey Roman, it's reproducable in terminal emulator called "konsole": 

https://konsole.kde.org/
https://www.archlinux.org/packages/extra/x86_64/konsole/

I tried latest commit in master but unfortunately it doesn't work too and now it says:

```
main.c:590 DEBUG:Got an event: SelectionRequest (30)
main.c:464 DEBUG:handling selection_request
main.c:515 DEBUG:We don't handle this target yet (0x1a4).
```

**Comment by kovetskiy** [comment]
We can try to write an integration test for konsole using keynav since we can spawn visual selection by clicking in konsole.

**Comment by RomanKovtyukh** [comment]
I don't shure if it's a good idea, to add some dependencies just for tests.

Little util that could emulate popular behaviour, will not only help us avoid new dependencies, it will even remove our current dependency of xclip.

**Comment by RomanKovtyukh** [comment]
Thank you, @seletskiy, I'll install konsole today on my machine and debug my implementation.

Today's morning I didn't pushed it because I didn't tested it yet. That's why it still doesn't work.

**Comment by RomanKovtyukh** [comment]
I fixed it in fbd7c8d960bdb64f5a60ee3586ca3e586c0a8ece.

But let keep this issue open while there is no tests for this situation.
